cli/src/main/java/org/aya/cli/render/RenderOptions.java

```diff
@@ -83,13 +119,15 @@ public boolean isDefault() {
   }
 
   public @NotNull StringStylist stylist(@NotNull OutputTarget output) throws IOException, JsonParseException {
-    if (isDefault()) return defaultStylist(output);
-    return switch (output) {
-      case Terminal -> new UnixTermStylist(buildColorScheme(), buildStyleFamily());
-      case LaTeX -> new TeXStylist(buildColorScheme(), buildStyleFamily());
-      case HTML -> new Html5Stylist(buildColorScheme(), buildStyleFamily());
-      case Plain -> new DebugStylist(buildColorScheme(), buildStyleFamily());
-    };
+    return stylistCaches.getOrPut(output, () -> Try.of(() -> {
+      if (isDefault()) return defaultStylist(output);
+      return switch (output) {
+        case Terminal -> new UnixTermStylist(buildColorScheme(), buildStyleFamily());
+        case LaTeX -> new TeXStylist(buildColorScheme(), buildStyleFamily());
+        case HTML -> new Html5Stylist(buildColorScheme(), buildStyleFamily());
+        case Plain -> new DebugStylist(buildColorScheme(), buildStyleFamily());
```

Comment:
It seems to build the same color scheme again when we require the stylist of another `OutputTarget`.

Reply:
Tql